sw/host/opentitantool/src/command/spx.rs

```diff
@@ -123,12 +123,12 @@ impl CommandDispatch for SpxSignCommand {
 
 #[derive(Debug, Args)]
 pub struct SpxVerifyCommand {
-    /// Key file.
+    /// The file containing the SPHINCS+ raw private key in PEM format.
     #[arg(value_name = "KEY")]
     key_file: PathBuf,
     /// Message file to verify the signature against.
     message: PathBuf,
-    /// SPHINCS+ signature file to verify.
+    /// SPHINCS+ signature file to verify, raw signature in PEM format.
```

Comment:
The signature is not in PEM format; it is just raw bytes.

Reply:
ah, I thought I looked at it and saw the base64 stuff, but I guess I mixed up, fixed.